po/fr.po

```diff
@@ -28346,122 +28408,119 @@ msgstr ""
 
 #: /tmp/fish/implicit/share/completions/grep.fish:2
 msgid "Process binary file as text"
-msgstr ""
+msgstr "Traiter les fichiers binaires comme du texte"
 
 #: /tmp/fish/implicit/share/completions/grep.fish:5
 msgid "Print byte offset of matches"
-msgstr ""
+msgstr "Afficher le décalage en octets des occurrences"
 
 #: /tmp/fish/implicit/share/completions/grep.fish:6
 msgid "Assume data type for binary files"
-msgstr ""
+msgstr "Présumer du type de données pour les fichiers binaires"
 
 #: /tmp/fish/implicit/share/completions/grep.fish:7
 msgid "Colour output"
-msgstr ""
+msgstr "Colorer la sortie"
 
 #: /tmp/fish/implicit/share/completions/grep.fish:8
 #: /tmp/fish/implicit/share/completions/jq.fish:10
 msgid "Color output"
-msgstr ""
+msgstr "Colorer la sortie"
 
 #: /tmp/fish/implicit/share/completions/grep.fish:9
 msgid "Only print number of matches"
-msgstr ""
+msgstr "N’afficher que le nombre d’occurrences"
 
 #: /tmp/fish/implicit/share/completions/grep.fish:10
-#, fuzzy
 msgid "Action for devices"
-msgstr "Afficher les dépendances importantes"
+msgstr "Action à effectuer sur les fichiers de périphérique"
 
 #: /tmp/fish/implicit/share/completions/grep.fish:11
-#, fuzzy
 msgid "Action for directories"
-msgstr "Nettoyer les répertoires source"
+msgstr "Action à effectuer sur les répertoires"
```

Comment:
Inconstitent terminology for directories: current translations almost exclusively use “dossiers”.

Reply:
I was confused because of the uses of *directory* and *folder*, and, as my translation work was very taylorist after the first hundred, I over-simplified by using *dossier* for *folder*, and *répertoire* for *directory*. Corrected.